SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -70,169 +70,142 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/**
-	 * Specifies the amount of time, in seconds, between heartbeat events sent from a game client to notify the server-worker instances
-	 * that it's connected.
+	 * From game clients to server-workers.
 	 */
-	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Interval (seconds)"))
+	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Interval (Seconds)"))
 	float HeartbeatIntervalSeconds;
 
-	/**
-	 * Specifies the maximum amount of time, in seconds, that the server-worker instances wait for a game client to send heartbeat events.
-	 * (If the timeout expires, the game client has disconnected.)
-	 */
-	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Timeout (seconds)"))
+
+	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Timeout (Seconds)"))
 	float HeartbeatTimeoutSeconds;
 
 	/**
-	 * Same as HeartbeatTimeoutSeconds, but used if WITH_EDITOR is defined.
+	 * Use when you are running the game server from the editor, or with the editor executable.
 	 */
-	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Timeout With Editor (seconds)"))
+	UPROPERTY(EditAnywhere, config, Category = "Heartbeat", meta = (DisplayName = "Heartbeat Timeout With Editor (Seconds)"))
 	float HeartbeatTimeoutWithEditorSeconds;
 
 	/**
-	 * Specifies the maximum number of Actors replicated per tick. Not respected when using the Replication Graph.
-	 * Default: `0` per tick  (no limit)
-	 * (If you set the value to ` 0`, the SpatialOS Runtime replicates every Actor per tick; this forms a large SpatialOS  world, affecting
-	 * the performance of both game clients and server-worker instances.) You can use the `stat Spatial` flag when you run project builds to
-	 * find the number of calls to `ReplicateActor`, and then use this number for reference.
+	 * Warning: default value 0 means that every Actor replicates. Ignored when using Replication Graph.
 	 */
-	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Maximum Actors replicated per tick"))
+	UPROPERTY(EditAnywhere, config, Category = "Replication", meta = (DisplayName = "Maximum Actors Replicated Per Tick"))
 	uint32 ActorReplicationRateLimit;
 
 	/**
-	 * Specifies the maximum number of entities created by the SpatialOS Runtime per tick. Not respected when using the Replication Graph.
-	 * (The SpatialOS Runtime handles entity creation separately from Actor replication to ensure it can handle entity creation requests
-	 * under load.) Note: if you set the value to 0, there is no limit to the number of entities created per tick. However, too many
-	 * entities created at the same time might overload the SpatialOS Runtime, which can negatively affect your game. Default: `0` per tick
-	 * (no limit)
+	 * Warning: default value 0 means there is no upper limit. Ignored when using Replication Graph.
```

Comment:
* Warning: default value 0 means there is no upper limit - this can affect the performance of your game. Note: Ignored when using Replication Graph.

//Same comment as above.